watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Sites/Site.java

```diff
@@ -71,7 +73,7 @@ public String toString() {
 
     @JsonIgnore
     public String getLocation() {
-        return String.format("%s, %s, %s %s", getStreet(), getCity(), getState(), getZipCode());
+        return String.format("%s\n%s, %s %d", getStreet(), getCity(), getState(), getZipCode());
```

Comment:
Rather than putting a newline in here, it would probably be better to just have 2 different textviews (one on top of the other). That'll give us finer control over the display as well, like having the street address be a different size or color than the rest of the location.